dags/probe_scraper.py

```diff
@@ -166,7 +166,7 @@
         endpoint=Variable.get("glean_dictionary_netlify_build_webhook_id"),
         method="POST",
         data={},
-        email=["bimsland@mozilla.com", "dataops+alerts@mozilla.com"],
+        email=["wlach@mozilla.com", "dataops+alerts@mozilla.com"],
```

Comment:
@wlach Are you the right owner/person to be notified here?

Reply:
I can fill in for now, for sure.